app/Models/Image.php

```diff
@@ -95,17 +96,7 @@ public static function upload(UploadedFile $file, $user, bool $forceReupload = f
 
         if ($image) {
             if ($forceReupload) {
-                // delete existing versions of the image
-                $filenames = scandir($image->getDirectory());
-                $imagePrefix = $image->id.'_';
-
-                $filenames = array_filter($filenames, function (string $filename) use ($imagePrefix) {
-                    return Str::startsWith($filename, $imagePrefix);
```

Comment:
This return statement's effect appears to have been lost in the refactoring of this code into the `clearExisting` function. While it is present inside that function, the new function's return value is never used. Did you deem it to be unnecessary?

Reply:
I'm not sure what you mean by this. `clearExisting` has no return value, and `upload` still returns an `Image` object. The original code has no return value in the first if block either.

Reply:
Ignore this comment; I somehow missed that line 103 is a return statement inside a lambda function. I am a silly pony.